packages/backend-core/src/middleware/authenticated.ts

```diff
@@ -115,7 +115,8 @@ export default function (
           authenticated = true
         } catch (err: any) {
           authenticated = false
-          console.error("Auth Error", err?.message || err)
+          console.error(`Auth Error: ${err.message}`)
+          console.error(err)
```

Comment:
Do we need this second error log?

Reply:
If we want to capture the stack trace yes, I was seeing log messages with just
```
"Auth Error": 
```